src/module/script.js

```diff
@@ -0,0 +1,44 @@
+import './index.js';
+
+let tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+const checkbox = document.querySelector('#myTasksList, #checkbox');
+
+const saveTasks = () => {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+};
+function checkCheckbox() {
+  tasks.forEach((task) => {
+    if (checkbox.checked) {
+      task.completed = false;
+    } else {
+      task.completed = true;
+    }
+  });
+  saveTasks();
+}
+
+checkbox.addEventListener('change', checkCheckbox);
```

Comment:
- [ ] As pointed out by the previous reviewer, the toggling of the completed state of the todo item does not work as expected. It does not persist in the localstorage
- [ ] Also, the add functionality also doesn't quite work well. Please kindly ensure all is fixed before requesting another review.